src/messaging/ReliableMessageMgr.cpp

```diff
@@ -154,6 +157,13 @@ void ReliableMessageMgr::ExecuteActions()
 
             // Do not StartTimer, we will schedule the timer at the end of the timer handler.
             mRetransTable.ReleaseObject(entry);
+
+            // That marked the exchange as not having an un-acked message... but
```

Comment:
What is "that" here?

Reply:
The ReleaseObject call.  I'll make that clearer.